packages/syntax-core/src/IconLinkButton/IconLinkButton.module.css

```diff
@@ -0,0 +1,36 @@
+@import url("../IconButton/IconButton.module.css");
```

Comment:
not rlly sure if this a css best-practice? I suppose I could alternatively import `IconButton.module.css` from inside `IconLinkButton.tsx` and use the styles there.

Reply:
i think you can just import the `IconButton` css file into IconLinkButton! that would make it so if any styles from IconButton change, it will change here too!